media/js/mozorg/contact-spaces.js

```diff
@@ -160,6 +160,7 @@
                     // Hide spaces and community nav+meta
                     mozMap.toggleNav('contact');
                     mozMap.toggleMobileNav('contact');
+                    mozMap.setMapState();
```

Comment:
Hmmmm, so I only see requests being made for spaces when it has not already been loaded. Are you seeing it always make a request to load the content or, are you referring to a different request?